README.md

```diff
@@ -73,6 +74,11 @@ the following keys:
   Indicates whether a test is enabled. Useful for disabling specific
   tests that are causing issues.
 
+- `requiresSdk`
```

Comment:
For compatibility with existing tests, have you thought about doing it the other way around: if there's no `requiresSdk` element, then assume that the SDK is required?

Also, what do you think about a more generic "requires" element? We know tests can require the "sdk". Maybe we would have some tests that dont even require the runtime? (Maybe self-contained tests?). Maybe some tests require something else from the host system? Or is a generic requires element too confusing?

Reply:
Generic one feels like it could get confusing really quickly while we don't have current use case.

And reverse compatibility isn't a problem (at least for me, and our regular-tests.)

Reply:
Fair enough.